docs/FAQ.md

```diff
@@ -223,6 +223,114 @@ Questions about VictoriaMetrics can be asked via the following channels:
 File bugs and feature requests [here](https://github.com/VictoriaMetrics/VictoriaMetrics/issues).
 
 
+### Where I can find information about Multi-Tenancy?
+
+You can read about Multitenancy [here](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#multitenancy). Note that only a [cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html) version supports Multi-Tenancy.
+
+
+
+### Will having multiple tenants affect my performance? Is this data written on disk separately or in one place?
+
+If you take the most challenging case  (multi-tenancy for the same set of metrics) it will require more disk space, more RAM for caches and more CPU for data processing. There should be linear growth for each additional tenant. But, in general, the overhead from additional tenants should be negligible because the metrics should be differentiated by labels. When the  tenant-id is added to the metric ID, and everything else - indexes, disk placement, remains the same.
```

Comment:
This does not sound clear to me when I read it. I expect, it doesn't for others too. First and last sentences needs to be rephrased